src/profile/profile.ts

```diff
@@ -681,6 +710,24 @@ export class Profile extends Logger {
     await this.ipld.remove(this.trees.bookmarkedDapps, `bookmarkedDapps/${address}`);
   }
 
+
+  /**
+   * Removes an identity.
+   *
+   * @param      {string}  address  The address
+   */
+  public async removeIdentityAccess(address: string): Promise<void> {
+    this.throwIfNotOwner('remove identity key');
+    const addressHash = this.nameResolver.soliditySha3(
+      ...[
+        this.nameResolver.soliditySha3(address),
+        this.nameResolver.soliditySha3(this.activeAccount),
+      ].sort(),
+    );
+    const addressBook = await this.getAddressBook();
+    delete (addressBook.keys[addressHash].identityAccess);
```

Comment:
This does not persist the changes to the address book, does it? You also need to store it again, no?

Reply:
I think the whole profile is built for handling just the data logic and developers can decide by their own, when the profile needs to be saved. As a result of this, multiple changes can be made with only one transaction.

Reply:
yup but this is in all cases of the profile the way ... you have to store it afterwards